SemiSpoilerLogger/UI/TrackerUI.cs

```diff
@@ -136,10 +137,24 @@ private void SetLabelText(int total)
             TextObject? label = layout?.GetElement<TextObject>("Area Tracker Label");
             if (label != null)
             {
-                label.Text = $"Major Items Remaining ({total})";
+                label.Text = $"{MainLabel()} Remaining ({total})";
             }
         }
 
+        private string MainLabel()
```

Comment:
I dislike noun-named functions, I think GetMainLabel would be more appropriate here